numerics/polynomial_evaluators_body.hpp

```diff
@@ -118,104 +120,150 @@ struct InternalEstrinEvaluator<Value, Argument, degree, low, 0> {
       ArgumentSquares const& argument_squares);
 };
 
-template<typename Value, typename Argument, int degree, int low, int subdegree>
-Derivative<Value, Argument, low>
-InternalEstrinEvaluator<Value, Argument, degree, low, subdegree>::Evaluate(
-    Coefficients const& coefficients,
-    Argument const& argument,
-    ArgumentSquares const& argument_squares) {
+template<typename Value,
+         typename Argument,
+         int degree,
+         int low,
+         int subdegree,
+         bool fma>
```

Comment:
Please pay attention to the formatting and try to stick to what we had before.  If all parameters don't fit on a single line, do `typename Value, typename Argument /n others`.  Same at the places where the template is instantiated.